Apps/Src/SendTritium.c

```diff
@@ -2,693 +2,777 @@
  * @copyright Copyright (c) 2018-2023 UT Longhorn Racing Solar
  * @file SendTritium.c
  * @brief Function implementations for the SendTritium application.
- * 
- * This contains functions relevant to updating the velocity and current setpoitns
- * of the Tritium motor controller. The implementation includes a normal current
- * controlled mode, a one-pedal driving mode (with regenerative braking), and cruise control.
- * The logic is determined through a finite state machine implementation.
- * 
- * If the macro SENDTRITIUM_EXPOSE_VARS is defined prior to including SendTritium.h, 
- * relevant setters will be exposed as externs for unit testing
- * and hardware inputs won't be read and motor commands won't be sent over MotorCAN.
- * If the macro SENDTRITIUM_PRINT_MES is also defined prior to including SendTritium.h,
- * debug info will be printed via UART.
+ *
+ * This contains functions relevant to updating the velocity and current
+ * setpoitns of the Tritium motor controller. The implementation includes a
+ * normal current controlled mode, a one-pedal driving mode (with regenerative
+ * braking), and cruise control. The logic is determined through a finite state
+ * machine implementation.
+ *
+ * If the macro SENDTRITIUM_EXPOSE_VARS is defined prior to including
+ * SendTritium.h, relevant setters will be exposed as externs for unit testing
+ * and hardware inputs won't be read and motor commands won't be sent over
+ * MotorCAN. If the macro SENDTRITIUM_PRINT_MES is also defined prior to
+ * including SendTritium.h, debug info will be printed via UART.
  */
 
-#include "Pedals.h"
-#include "ReadCarCAN.h"
-#include "Minions.h"
 #include "SendTritium.h"
+
+#include "CanBus.h"
+#include "CanConfig.h"
+#include "Minions.h"
+#include "Pedals.h"
+#include "ReadCarCan.h"
 #include "ReadTritium.h"
-#include "SendCarCAN.h"
-#include "CANbus.h"
+#include "SendCarCan.h"
 #include "UpdateDisplay.h"
-#include "CANConfig.h"
 #include "common.h"
 
 // Macros
-#define MAX_VELOCITY 20000.0f // rpm (unobtainable value)
+#define MAX_VELOCITY 20000.0f  // rpm (unobtainable value)
 
-#define MIN_CRUISE_VELOCITY mpsToRpm(20.0f) // rpm
-#define MAX_GEARSWITCH_VELOCITY mpsToRpm(8.0f) // rpm
+#define MIN_CRUISE_VELOCITY MpsToRpm(20.0f)     // rpm
+#define MAX_GEARSWITCH_VELOCITY MpsToRpm(8.0f)  // rpm
 
-#define BRAKE_PEDAL_THRESHOLD 15  // percent
-#define ACCEL_PEDAL_THRESHOLD 10 // percent
+#define BRAKE_PEDAL_THRESHOLD 50  // percent
```

Comment:
I don't have a good understanding of how the brake feels/works, but is 50 relatively high for a threshold?

Reply:
Idk why either but that is what made it into M2 release, so it should be fine. @IshDeshpa know why that changed? I think it was because we kept going into brake state or something but I think we solved that with the numbers in Pedals

Reply:
It's because brake pedal is just a button I believe.